tests/core/pthread/test_pthread_dylink_tls.out

```diff
@@ -0,0 +1,15 @@
+thread: 1
+side_tls value  : 42
+main_tls value  : 10
+increment done
+side_tls value  : 62
+main_tls value  : 30
+
+thread: 2
+side_tls value  : 42
+main_tls value  : 10
```

Comment:
`main_tls` begins at 10, then is bumped by 20 to 30. But why is it back to 10 here?

Reply:
Oh, that is inside the thread... indeed confusing all this stuff.